mattermost-helm/charts/mattermost-app/templates/jobserver-deployment.yaml

```diff
@@ -13,11 +13,20 @@ spec:
         app: {{ template "fullname" . }}-jobserver
         component: "{{ .Values.global.features.jobserver.name }}"
     spec:
+      initContainers:
+        - name: "init-elasticsearch"
+          image: "appropriate/curl:latest"
+          imagePullPolicy: "IfNotPresent"
+          command: ["sh", "-c", "until ! {{ .Values.global.features.elasticsearch.enabled }} || curl --max-time 5 http://{{ .Release.Name }}-mattermost-elasticsearch:9200 ; do echo waiting for elasticsearch; sleep 5; done; echo init-elasticsearch finished"]
+        - name: "init-mysql"
+          image: "appropriate/curl:latest"
+          imagePullPolicy: "IfNotPresent"
+          command: ["sh", "-c", "until curl --max-time 5 http://{{ .Release.Name }}-mysqlha-readonly:3306; do echo waiting for {{ .Release.Name }}-mysqlha; sleep 5; done;"]
       containers:
       - name: {{ template "mattermost-app.jobserver.fullname" . }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        command: ["platform", "jobserver"]
+        command: ["mattermost", "jobserver"]
```

Comment:
I don't think we can switch this yet, some of the images will still be pre-5.0 for a while

Reply:
since we are using the `mattermost/mattermost-enterprise-edition` and this only contains images >5.0.0-rc1 i guess we can set. but if you want i can revert no worries from my side

Reply:
We will still need to load test old versions (especially 4.10 which is the ESR). I've already added  tag to `mattermost/mattermost-enterprise-edition` for 4.10.1 because the ltops/kubes tool uses that

Reply:
ok!